eng/Signing.props

```diff
@@ -4,6 +4,9 @@
     <!-- The FullNugetVersion.version file only gets produced in the initial build pass. -->
     <RelativeBlobPathParent Condition="'$(DotNetBuildPass)' == '' or '$(DotNetBuildPass)' == '1'">Sdk/$([System.IO.File]::ReadAllText('$(ArtifactsTmpDir)FullNugetVersion.version').Trim())</RelativeBlobPathParent>
     <PublishBinariesAndBadge Condition="'$(PublishBinariesAndBadge)' == ''">true</PublishBinariesAndBadge>
+
+    <HasVSInsertionArtifact Condition="'$(BuildWorkloads)' == 'true' or ('$(DotNetBuildPass)' == '2' and '$(OS)' == 'Windows_NT' and '$(Architecture)' == 'x64')">true</HasVSInsertionArtifact>
+    <VSInsertionArtifactRelativeBlobPathParent Condition="'$(HasVSInsertionArtifact)' == 'true'">VSInsertion/$([System.IO.File]::ReadAllText('$(ArtifactsTmpDir)VSInsertionVersion.version').Trim())</VSInsertionArtifactRelativeBlobPathParent>
```

Comment:
Move to the same pattern in `Publishing.props` now that we don't do this pattern for the regular sdk blobs?